dependencies.yaml

```diff
@@ -124,32 +124,44 @@ dependencies:
         matrices:
           - matrix:
               cuda: "11.8"
-              arch: x86_64
             packages:
               - cudatoolkit=11.8
-              - libcufile=1.4.0.31
-              - libcufile-dev=1.4.0.31
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+          - matrix:
+              cuda: "11.4"
+            packages:
+              - cudatoolkit=11.4
+          - matrix:
+              cuda: "11.2"
+            packages:
+              - cudatoolkit=11.2
+      - output_types: conda
+        matrices:
           - matrix:
               cuda: "11.8"
-              arch: aarch64
+              arch: x86_64
             packages:
-              - cudatoolkit=11.8
+              - libcufile=1.4.0.31
```

Comment:
Can you add a comment linking to the page that enumerates these version numbers?

Reply:
This is currently scraped from https://anaconda.org/nvidia/libcufile/files but we'll be using https://anaconda.org/conda-forge/libcufile for CUDA 12 versions. I can add a comment for now but it'll need to be updated later.

Reply:
```suggestion
              # libcufile package version reference: https://anaconda.org/nvidia/libcufile/files
              - libcufile=1.4.0.31
```